test/curl/setOpt.spec.ts

```diff
@@ -57,6 +57,11 @@ describe('setOpt()', () => {
     curl.setOpt('WRITEFUNCTION', null)
   })
 
+  it('should be able to set blob value back to null', () => {
+    curl.setOpt('SSLKEY_BLOB', Buffer.from([]))
+    curl.setOpt('SSLKEY_BLOB', null)
+  })
```

Comment:
Can you add another 2 test cases for:

- String value
- Buffer with data (the above one is empty)

Reply:
Done.